src/utils/EmailValidator.js

```diff
@@ -0,0 +1,18 @@
+
+const IsEmailValid = (emailValue) => {
```

Comment:
Function names in JavaScript don't start with a capital letter unless it's a  React component with some other exceptions.